tests/qa_automation/execute_test_run.pm

```diff
@@ -33,7 +34,7 @@ sub run {
     save_screenshot;
 
     #output result to serial0 and upload test log
-    if (get_var("QA_TESTSUITE")) {
+    if (get_var("QA_TESTSUITE") && !is_jeos) {
```

Comment:
Could you please at least add a comment why we don't want to do this step on jeos?

Reply:
Updated